src/runtime_src/core/pcie/driver/linux/xocl/xocl_fdt.c

```diff
@@ -196,7 +196,7 @@ static void devinfo_cb_plp_gate(void *dev_hdl, void *subdevs, int num)
 {
 	struct xocl_subdev *subdev = subdevs;
 
-	subdev->info.level = XOCL_SUBDEV_LEVEL_BLD;
```

Comment:
Based on my test, it should be PRP, please let me know if you think it should be BLD.